operations/configure_cray_cli.md

```diff
@@ -21,50 +25,195 @@ The `cray` CLI only needs to be initialized once per user on a node.
    ncn# unset CRAY_CREDENTIALS
    ```
 
-1. Initialize the `cray` CLI for the root account.
+1. Initialize the `cray` CLI for the root account on all master and worker nodes.
 
-   The `cray` CLI needs to know what host to use to obtain authorization and what user is requesting authorization,
-   so it can obtain an OAUTH token to talk to the API Gateway. This is accomplished by initializing the CLI
-   configuration. In this example, the 'vers' username and its password are used.
+   The script will handle creation of the temporary keycloak user and initialize all master and
+   worker nodes that are in a ready state.  Call the script with the run option:
```

Comment:
We're supposed to single space between sentences in the docs, so I implore you to remove the surplus space between these two sentences.